### PR TITLE
Introduce the guide at the beginning, not part way through

### PR DESCRIPTION
**Pull request by adammil2000**
Getting started guide doesn't explain itself and mention the demo profile until part way through.

### CODE REVIEW

File: content/en/docs/setup/getting-started/index.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ aliases:
 keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
+This guide is intended for users who are new to Istio and provide streamlined instruction to
+install Istio's built-in `demo` [configuration profile](/docs/setup/additional-setup/config-profiles/).
+The `demo` installation lets you quickly get started evaluating Istio.
```
Comment (ericvn) [inline content/en/docs/setup/getting-started/index.md]: c/`demo` installation/`demo` configuration profile/ possibly. The later is used on line 21.

Comment (ericvn) [inline content/en/docs/setup/getting-started/index.md]:
```suggestion
The `demo` configuration profile lets you quickly get started evaluating Istio.
```

Comment (adammil2000) [reply content/en/docs/setup/getting-started/index.md]: fixed

Comment (adammil2000) [reply content/en/docs/setup/getting-started/index.md]: Fixed

**Pull request merged**